### DISCUSSION TITLE
Model Titles

### DISCUSSION THREAD

**Discussion by nisia97** [keras-team/autokeras#1462] [Q&A]
Hello,

I am new to autokeras and am running into an issue recreating a model that is determined by autokeras trials.

Layer (type)                 Output Shape              Param #   
=================================================================
input_1 (InputLayer)         [(None, 3)]               0         
_________________________________________________________________
multi_category_encoding (Mul (None, 3)                 0         
_________________________________________________________________
normalization (Normalization (None, 3)                 7         
_________________________________________________________________
dense (Dense)                (None, 1024)              4096      
_________________________________________________________________
re_lu (ReLU)                 (None, 1024)              0         
_________________________________________________________________
dense_1 (Dense)              (None, 64)                65600     
_________________________________________________________________
re_lu_1 (ReLU)               (None, 64)                0         
_________________________________________________________________
dropout (Dropout)            (None, 64)                0         
_________________________________________________________________
regression_head_1 (Dense)    (None, 1845)              119925    
=================================================================
Total params: 189,628
Trainable params: 189,621
Non-trainable params: 7
_________________________________________________________________

Here, I am not sure specifically what kind of layer is supposed to be a multi_category_encoding layer or a Noemalization layer as these are not options when building a model with keras. I have tried to load the model and train it where it left off, but that does not work either. Any help is greatly appreciated!

**Comment by haifeng-jin** [comment]
AutoKeras use some custom layers https://github.com/keras-team/autokeras/blob/master/autokeras/keras_layers.py
